config/collectorConfig.js

```diff
@@ -71,6 +74,17 @@ module.exports = {
    */
   heartbeatLatencyToleranceMillis,
 
+  /*
+   * The number of upserts a generator can miss before it will be assumed to be
+   * stuck and reassigned to a different collector.
+   */
+  generatorUpsertToleranceFactor,
```

Comment:
Are these new config items intended to be sent to the running refocus collectors? I think the config items in this file are config items which can be defined or overridden by Refocus but are sent to the collectors to alter the behavior of the collectors